recipes/gcc/gcc-common.inc

```diff
@@ -229,6 +229,7 @@ do_configure_gcc() {
 	--disable-multilib \
 	--disable-multiarch \
 	--with-system-zlib \
+	--disable-libsanitizer \
```

Comment:
How do you plan to get around this?  You have marked the commit disabling libsanitizer with "THIS IS ONLY FOR TESTING WITH MUSL"...